src/classes/ERR_Handler.cls

```diff
@@ -138,62 +138,155 @@ public class ERR_Handler {
     	}
     	return errors;
     }
-        
+
+	/*******************************************************************************************************
+    * @description Processes the results of a group DML operations caused by a batch, future, or queued job and performed
+    * 				using the Database class to determine if errors occurred. It does not store errors or send error email
+    * 				notifications. These operations should be performed after the rollback.
+    *
+    * @param dmlResults The results of a DML batch operation performed using the Database class.
+    * @param sObjects The records the DML operation was performed on.
+    * @return Errors The errors with a single list that occurred during a batch DML operation performed using the Database class.
+    */
+	public static Errors getJobErrors(List<Object> dmlResults, List<SObject> sObjects) {
+		Errors errors = new Errors();
+		List<String> errorsFound = new List<String>();
+
+		if(dmlResults != null) {
```

Comment:
our coding standard is a space after if and for, so it should be if (...) and for (...)

Reply:
Cool, good to know! I'll make the necessary edits on those.